react-native-cli/index.js

```diff
@@ -136,12 +138,12 @@ function createProject(name) {
   fs.writeFileSync(path.join(root, 'package.json'), JSON.stringify(packageJson));
   process.chdir(root);
 
-  console.log('Installing react-native package from npm...');
-  exec('npm install --save react-native', function(e, stdout, stderr) {
+  console.log('Installing ' + REACT_NATIVE_PACKAGE_ID + ' package from npm...');
+  exec('npm install --save ' + REACT_NATIVE_PACKAGE_ID, function(e, stdout, stderr) {
```

Comment:
I'll let @frantic and @foghina make the call since they can make the call from FB's security perspective. I didn't test with cli.js but I was able to inject code into a similar child_process.exec with `HACK="&& ls" node test.js`.